### PR TITLE
Adding tips to the README.md

### PR DESCRIPTION
**Pull request by jbianquetti-nami**

### CODE REVIEW

**Comment by sameersbn** [comment]
Requesting @vikram-bitnami for a review

File: docs/quickstart-gke.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This document walks you through setting up an Google Kubernetes Engine (GKE) cluster and installing the Bitnami Kubernetes Production Runtime (BKPR) to that cluster.
+This document walks you through setting up a Google Kubernetes Engine (GKE) cluster and installing the Bitnami Kubernetes Production Runtime (BKPR) to that cluster.
```
Comment (vikram-bitnami) [inline docs/quickstart-gke.md]: to that -> on that

File: docs/quickstart-gke.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ In this section, you will deploy an Google Kubernetes Engine (GKE) cluster using
...
+  - `GCLOUD_AUTHZ_DOMAIN` specifies the email domain of authorized users and needs to be a [G Suite](https://gsuite.google.com/) domain. Alternatively, you could specify the value `*`, but be __WARNED__ that this will authorize any Google account holder to authenticate with OAuth to your cluster services.
   - `GCLOUD_K8S_CLUSTER` specifies the name of the GKE cluster.
   - `GCLOUD_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR. `gcloud container get-server-config --project ${GCLOUD_PROJECT} --zone ${GCLOUD_REGION}` lists the versions available in your region.
 
-* Create a OAuth Client ID by following these steps
+* Create an OAuth Client ID by following these steps
 
   1. Go to https://console.developers.google.com/apis/credentials
```
Comment (vikram-bitnami) [inline docs/quickstart-gke.md]: Make the link clickable?

File: docs/quickstart-gke.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ In this section, you will deploy an Google Kubernetes Engine (GKE) cluster using
   5. Add the TLD of the domain specified in the `GCLOUD_DNS_ZONE` variable to the __Authorized domains__ and save the changes
   6. Choose __Credentials__ tab and select the __Create Credentials > OAuth client ID__
```
Comment (vikram-bitnami) [inline docs/quickstart-gke.md]: Choose the

File: docs/quickstart-gke.md [hunk 4]
```diff
@@ -34,11 +34,11 @@ In this section, you will deploy an Google Kubernetes Engine (GKE) cluster using
...
-  - `GCLOUD_AUTHZ_DOMAIN` specifies the email domain of authorized users and needs to be a [G Suite](https://gsuite.google.com/) domain. Alternatively you could specify the value `*`, but be __WARNED__ that this will authorize any Google account holder to authenticate with OAuth to your cluster services.
+  - `GCLOUD_AUTHZ_DOMAIN` specifies the email domain of authorized users and needs to be a [G Suite](https://gsuite.google.com/) domain. Alternatively, you could specify the value `*`, but be __WARNED__ that this will authorize any Google account holder to authenticate with OAuth to your cluster services.
   - `GCLOUD_K8S_CLUSTER` specifies the name of the GKE cluster.
   - `GCLOUD_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR. `gcloud container get-server-config --project ${GCLOUD_PROJECT} --zone ${GCLOUD_REGION}` lists the versions available in your region.
 
-* Create a OAuth Client ID by following these steps
+* Create an OAuth Client ID by following these steps
```
Comment (vikram-bitnami) [inline docs/quickstart-gke.md]: steps:

File: docs/quickstart-gke.md (hunk 2 above)
Comment (vikram-bitnami) [inline docs/quickstart-gke.md]: End this and subsequent sentences with periods.

File: docs/quickstart-gke.md (hunk 1 above)
Comment (jbianquetti-nami) [reply docs/quickstart-gke.md]: Done.

File: docs/quickstart-gke.md (hunk 2 above)
Comment (jbianquetti-nami) [reply docs/quickstart-gke.md]: Done

File: docs/quickstart-gke.md (hunk 3 above)
Comment (jbianquetti-nami) [reply docs/quickstart-gke.md]: Done

File: docs/quickstart-gke.md (hunk 4 above)
Comment (jbianquetti-nami) [reply docs/quickstart-gke.md]: Done

File: docs/quickstart-gke.md (hunk 2 above)
Comment (jbianquetti-nami) [reply docs/quickstart-gke.md]: Done

**Pull request merged**